tajo-core-tests/src/test/resources/results/TestTajoCli/testHelpSessionVars.result

```diff
@@ -42,4 +42,5 @@ Available Session Variables:
 \set ARITHABORT [true or false] - If true, a running query will be terminated when an overflow or divide-by-zero occurs.
 \set FETCH_ROWNUM [int value] - Sets the number of rows at a time from Master
 \set BLOCK_ON_RESULT [true or false] - Whether to block result set on query execution
+\set RESULT_COMPRESS [true or false] - Enable resultSet compression
```

Comment:
I have some comments.

* ``RESULT_COMPRESSION`` would be better in terms of naming convention.
* Probably, ``RESULT_COMPRESS`` and its description may seem to enable the compressed query result. 
* Actually, this feature compresses the transfer stream for ResultSet. So, I'd like to suggest the description as ``Use compression to optimize result transmission." 
* Also, I'd like to suggest this key ``COMPRESSED_RESULT_TRENSFER`` for session variable. It's clear, but it is somewhat longer than yours. It's up to you.

Also, I'm going to improve JDBC URL to take defined options instead of taking directly session variables. For compressed data transfer, I'll add ``?useCompression=true`` like MySQL.